jenkins_jobs/linux.yml

```diff
@@ -54,18 +54,15 @@
     builders:
       # Setting BUILD_ID for the vagrant process to make sure the Jenkins process tree killer
       # doesn't kill the VM before the next job is started.
-      #
-      # Since the node_modules directory might not be synchronized between VM and host (GPII-2060)
-      # it's necessary to run `npm install` on the host to get auxiliary tools installed (e.g grunt)
-      - shell: BUILD_ID=gpii-linux DISPLAY=:0 vagrant up --provider virtualbox && npm install
+      - shell: BUILD_ID=gpii-linux DISPLAY=:0 vagrant up --provider virtualbox
```

Comment:
also the `DISPLAY=:0` is no longer needed since that environment variable is already set when a new terminal is opened:
https://github.com/idi-ops/packer-fedora/blob/master/scripts/user-envs.sh

Reply:
Do you mean it's site in a new terminal inside the VM?

I get a failure if I don't set it on the host so VirtualBox knows where to open the GUI on.

```
[gtirloni@h-0005 universal]$ echo $DISPLAY

[gtirloni@h-0005 universal]$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'inclusivedesign/fedora27'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'inclusivedesign/fedora27' is up to date...
==> default: Setting the name of the VM: universal_default_1522346127284_17025
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 8081 (guest) => 8081 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "223199a0-657b-4fd3-8823-dc31e6a94475", "--type", "gui"]

Stderr: VBoxManage: error: The virtual machine 'universal_default_1522346127284_17025' has terminated unexpectedly during startup because of signal 6
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MachineWrap, interface IMachine
```